### ISSUE TITLE
Related sentences (instead of notes) to current sentence (or block level)

### ISSUE THREAD

**Issue by vasnt** [reorproject/reor#153]
**Is your feature request related to a problem? Please describe.**
When notes are non-atomic (mixed) type, it is better to have similarity suggestions with respect to current/each sentence. 

**Describe the solution you'd like**
Add feature to show similarity suggestions with respect to current/each sentence. 

**Additional context**

**Comment by samlhuillier** [comment]
Yes I agree with this. How would you propose this would work? Would it be like where the cursor is is what is used to show similarity? Or like highlight a block of text and just show similarity for that?

**Comment by vasnt** [comment]
I prefer, it should read current sentence (instead of just current word at cursor) & show all related sentences from other notes. 
In fact it can also keep context of header/paragraph title(if any) alongwith current sentence. But primarily current sentence.

**Comment by samlhuillier** [comment]
Yes by cursor I meant, it would use either the current sentence or block of where the cursor is. And I agree that then users should be able to optionally alter the context that is used for similarity search by highlighting
